public/app/features/canvas/runtime/element.tsx

```diff
@@ -489,11 +489,14 @@ export class ElementState implements LayerElement {
   };
 
   applyRotate = (event: OnRotate) => {
-    const absoluteRotationDegree = event.absoluteRotation;
-
+    const rotationDelta = event.delta;
     const placement = this.options.placement!;
+    const placementRotation = placement.rotation ?? 0;
+
+    const calculatedRotation = placementRotation + rotationDelta;
```

Comment:
This change impacts both single element and group element rotation; TLDR by applying the relative delta we can have a solution that works for both single element + group of element rotation (see commit history for additional context if interested)